spark/src/test/java/org/opensearch/sql/spark/asyncquery/IndexQuerySpecAlterTest.java

```diff
@@ -70,7 +70,7 @@ public GetJobRunResult getJobRunResult(String applicationId, String jobId) {
               MockFlintSparkJob flintIndexJob =
                   new MockFlintSparkJob(
                       flintIndexStateModelService, mockDS.getLatestId(), MYS3_DATASOURCE);
-              flintIndexJob.active();
+              flintIndexJob.refreshing();
```

Comment:
Changing these because
1. Auto refresh index's normal state is `REFRESHING`. It shouldn't be `ACTIVE`
2. This PR introduce a change where cancelStreamingJob will only be called if index state is initially `REFRESHING`